tests/integration/test_broken_projections/config.d/backups.xml

```diff
@@ -9,5 +9,6 @@
     </storage_configuration>
     <backups>
         <allowed_disk>backups</allowed_disk>
+        <test_randomize_order>true</test_randomize_order>
```

Comment:
When I added setting `backups.test_randomize_order` to the test configuration it made the issue much easier to reproduce.